x/epochs/genesis_test.go

```diff
@@ -12,8 +12,6 @@ import (
 
 func TestGenesis(t *testing.T) {
 	genesisState := types.GenesisState{
-		Params: types.DefaultParams(),
```

Comment:
why removing params?

Reply:
I dont think those params are set at genesis for the epochs module like they are in stakeibc

StakeIBC Ref
https://github.com/Stride-Labs/stride/blob/main/x/stakeibc/genesis.go#L37
https://github.com/Stride-Labs/stride/blob/main/x/stakeibc/genesis.go#L43